ROVU/Core/src/utility/TestSuite.java

```diff
@@ -53,32 +68,181 @@ public void robotPositioningTest() {
 		}
 		
 		@Test
+		/**
+		 * Test #2: Robot's movement to two points
+		 */
 		public void robotMoveTest_1() {
```

Comment:
Why is this entire test commented out?

Reply:
This one is obsolete, will be updated when we get the robotcontroller to move robots.